docs/service_binding.adoc

```diff
@@ -0,0 +1,85 @@
+== Service Binding 
+
+When you are writing an application that requires multiple services (such as Kafka and a Service Registry), you normally have to configure connections to these services by hand in a configuration file. These connections often include public data such as well known urls as well as private data such as passwords and account IDs. Custom resources managed by the RHOAS operator provide service binding annotations which allows the link:{https://redhat-developer.github.io/service-binding-operator/userguide/intro.html}[Service Binding Operator (SBO)] to expose this connection information to your application. Because the SBO manages this information it can stay securely inside of your Open Shift cluster instead of being shared around in configuration files in source control.
```

Comment:
```suggestion
When you are writing an application that requires multiple services (such as Kafka and a Service Registry), you normally have to configure connections to these services by hand in a configuration file. These connections often include public data such as well known urls as well as private data such as passwords and account IDs. Custom resources managed by the RHOAS operator provide service binding annotations which allows the link:{https://redhat-developer.github.io/service-binding-operator/userguide/intro.html}[Service Binding Operator (SBO)] to expose this connection information to your application. Because the SBO manages this information it can stay securely inside of your Open Shift cluster instead of being shared across different configuration files in source control.
```